lib/cartopy/tests/mpl/__init__.py

```diff
@@ -270,3 +275,62 @@ def image_as_base64(fname):
 
     html.extend(['</body>', '</html>'])
     return '\n'.join(html)
+
+
+def show(projection, geometry):
+    orig_backend = matplotlib.get_backend()
+    plt.switch_backend('tkagg')
```

Comment:
It's not really used (just in comments), so I suppose you can do away with it as you please.